### PR TITLE
certigo 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by christodenny**
Certigo is a utility to examine and validate certificates
in a variety of formats.

### CODE REVIEW

File: Formula/certigo.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Certigo < Formula
+  desc "Utility to examine and validate certificates in a variety of formats"
+  homepage "https://github.com/square/certigo"
+  url "https://github.com/square/certigo.git"
```
Comment (bfontaine) [inline Formula/certigo.rb]: We need a tagged release here.

File: Formula/certigo.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    system "./build"
+    bin.install "bin/certigo"
+  end
+
+  test do
+    system "certigo", "help"
```
Comment (bfontaine) [inline Formula/certigo.rb]: Please use `"#{bin}/certigo"`. Also, can’t we write a test that does something more substantial?

Comment (christodenny) [inline Formula/certigo.rb]: Anything other than help or version requires a valid tls certificate as input which is hard to do in a ruby script.

Comment (bfontaine) [inline Formula/certigo.rb]: Can’t we e.g. include a dummy certificate then ensure `certigo` says it’s invalid?

Comment (christodenny) [inline Formula/certigo.rb]: Updated @bfontaine

File: Formula/certigo.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
+class Certigo < Formula
+  desc "Utility to examine and validate certificates in a variety of formats"
+  homepage "https://github.com/square/certigo"
+  url "https://github.com/square/certigo.git",
+    :tag => "v1.1.0",
+    :revision => "beac5ca8f48521a8361df8c953e66902f1d6632c"
+  sha256 "1ca8ee1130d57fb70b1c21cc9311d8b036b86d456eb88888d603e88fbb1d056a"
```
Comment (DomT4) [inline Formula/certigo.rb]: Can't shasum a git checkout. The `:revision` tag here serves as the checksum, so you can remove this `sha256` line.

File: Formula/certigo.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    system "./build"
+    bin.install "bin/certigo"
+  end
+
+  test do
+    (testpath/"test.crt").write("-----BEGIN CERTIFICATE-----
```
Comment (DomT4) [inline Formula/certigo.rb]: Please use `.write <<-EOS.undent`  here and then indent appropriately. Looks a bit cleaner.

File: Formula/certigo.rb [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+oX6eEpKIo/QtB6mSU216F/PuAzl/BJond+RzF9mcxJjdZYZlhwT8+o8oXEMI4vEf
+3yzd+zB/mjuxDJR2iw3bSL+zZr2GV/CsMLG/jmvbpIuyI/p5eTy0alz+iHOiyeCN
+9pgD6jyonwIDAQABMA0GCSqGSIb3DQEBCwUAA4IBAQAMUuv/zVYniJ94GdOVcNJ/
+bL3CxR5lo6YB04S425qsVrmOex3IQBL1fUduKSSxh5nF+6nzhRzRrDzp07f9pWHL
+ZHt6rruVhE1Eqt7TKKCtZg0d85lmx5WddL+yWc5cI1UtCohB9+iZDPUBUR9RcszQ
+dGD9PmxnPc9soEcQw/3iNffhMMpLRhPaRW9qtJU8wk16DZunWR8E0Oeq42jVTnb4
+ZiD1Idajj0tj/rT5/M1K/ZLEiOzXVpo/+l/+hoXw9eVnRa2nBwjoiZ9cMuGKUpHm
+YSv7SyFevNwDwcxcAq6uVitKi0YCqHiNZ7Ye3/BGRDUFpK2IASUo8YbXYNyA/6nu
+-----END CERTIFICATE-----")
+    system "#{bin}/certigo", "dump", "test.crt"
```
Comment (DomT4) [inline Formula/certigo.rb]: I'll get glared at by ILZ if I don't point out there's a minor preference towards `bin/"certigo"`, but it's only a style thing.

File: Formula/certigo.rb (hunk 4 above)
Comment (christodenny) [inline Formula/certigo.rb]: I initially used that format but it failed the jenkins build. After changing it to this it started passing. Could that fail be a one off thing and I should just try using EOS.undent anyways? @DomT4

File: Formula/certigo.rb (hunk 3 above)
Comment (christodenny) [inline Formula/certigo.rb]: Fixed @DomT4

File: Formula/certigo.rb (hunk 4 above)
Comment (bfontaine) [inline Formula/certigo.rb]: Jenkins had some issues today, you should be fine now.

Comment (christodenny) [inline Formula/certigo.rb]: Updated @bfontaine @DomT4

**Comment by DomT4** [comment]
Thanks! Looks pretty good. Would you possibly mind squashing this down to one commit again? Saves one of us having to do it on merge 🙈.

**Comment by christodenny** [comment]
Commits have been squashed. @DomT4 @bfontaine

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/01fcc2e3226e165d07204dd36d116fdfeca931e3. Thank you for your contribution to Homebrew @christodenny; we appreciate it! :smiley_cat:

**Pull request closed without merging**